tests/client/main.cpp

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define CATCH_CONFIG_MAIN
+
+#include "header.h"
+#include "catch2\catch.hpp"
+#include "rpc_interface_h.h"
+#pragma warning(push)
+#pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
+#pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
+                                // 'type2', possible loss of data
+#undef VOID
+#include "ebpf_verifier.hpp"
+#pragma warning(pop)
+#include "ebpf_windows.h"
+#include "ebpf_api.h"
+#include "service_helper.h"
+
+#define EBPF_SERVICE_BINARY_NAME L"ebpfsvc.exe"
+#define EBPF_SERVICE_NAME L"ebpfsvc"
+
+RPC_STATUS
+initialize_rpc_binding();
```

Comment:
if these are internal to this file, make them static.
If declared in another file, move prototypes to a .h file.